src/Package/Impl/Repl/Commands/ReplShortcutSetting.cs

```diff
@@ -2,91 +2,39 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Diagnostics;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
 using EnvDTE;
-using Microsoft.R.Editor.Settings;
 using Microsoft.VisualStudio.R.Package.Shell;
 
 namespace Microsoft.VisualStudio.R.Package.Repl.Commands {
     internal static class ReplShortcutSetting {
-        private const string CommandName = "EditorContextMenus.CodeWindow.ExecuteLineInInteractive";
-
-        private static bool _currentSetting;
+        private static Dictionary<string, string> _editorShortcutRemaps = new Dictionary<string, string>() {
+            { "Edit.LineOpenAbove", "Text Editor::ctrl+k,ctrl+o" }, // Make Ctrl+Enter work in R Editor
```

Comment:
It is interesting question. I can try and check what is current setting (maybe Vim extensions moves it...)

I have a thread with IDE folks and it looks like core editor ignores narrower definitions and still uses its own :-(. I am going to open a bug on this for them in Dev15.